### PR TITLE
www: citing.html: fix escaping

### LINKED ISSUE #2611: Fix bibtex entry in citing.html
#2594 added `citing.html`, which has a bibtex entry that is rendered as follows:
```
@article{aspectmanual, 
title = {{	extsc{ASPECT}: Advanced Solver for Problems in Earth's ConvecTion, User Manual}}, 
author = {W. Bangerth and J. Dannberg and R. Gassm{"o}ller and T. Heister and others}, 
```
That's missing backslashes and text in `\testsc` and `"o`.

### PR DESCRIPTION
**Pull request by tjhei**
fixes #2611
also see #2662

### CODE REVIEW

File: database.js [hunk 1]
```diff
@@ -1,51 +1,54 @@
+// Auto-generated file by https://github.com/geodynamics/aspect/doc/make_cite_html.py
```
Comment (jdannberg) [inline database.js]: That link doesn't work for me. Although it's clear what you mean, maybe either put the full link https://github.com/geodynamics/aspect/blob/master/doc/make_cite_html.py, or just say aspect/doc/make_cite_html.py.

Comment (tjhei) [reply database.js]: fixed

**Comment by jdannberg** [comment]
Actually, the comment I had is still not fixed. The plain text list of publications is missing now [on the citing website](https://aspect.geodynamics.org/citing.html), and instead there's just an empty box.

**Comment by tjhei** [comment]
> Can you check what's going on there?

argh. Let me check.

**Comment by tjhei** [comment]
> I opted for a voluntary root canal operation instead

oh, the file is automatically generated, you can just convince yourself that the script is changed in a sensible way in #2664

**Pull request merged**